backend-core-model/src/main/java/com/flowingcode/backendcore/model/constraints/AttributeConstraint.java

```diff
@@ -21,6 +21,11 @@
 
 import com.flowingcode.backendcore.model.Constraint;
 
+/**
+ * Marker interface for constraints applied to entity attributes.
```

Comment:
A marker interface is an interface with no methods or fields.

This would be a "Specialization of `{@code Constraint}` that applies to entity attributes."